packages/graphql/src/services/accounts.ts

```
@@ -1088,12 +1089,24 @@ export async function changeEmailSettings(
+  const changeEmailSettingsTask = changeUserEmailSettings(ctx.hatchet)
+  const { success } = await changeEmailSettingsTask.run({
+    userId: ctx.user.sub,
+    projectUpdates,
+  })
+
+  if (!success) {
+    return user
+  }
+  {
+    return { ...user, sendProjectUpdates: projectUpdates }
+  }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)